docs/program/SDK-as-client.md

```diff
@@ -186,6 +192,23 @@ main();
 {{% /tab %}}
 {{< /tabs >}}
 
+{{% alert title="Caution" color="caution" %}}
+
+Do not share your robot secret or robot address publicly.
+Sharing this information compromises your system security by allowing unauthorized access to your computer.
+
+{{% /alert %}}
+
+## Run Your Code
+
+After saving the above code and adding control logic with Viam's API methods, run your program to control your Viam-connected robot.
+
+For example:
+
+- Python: `python3 ~/myCode/myViamFile.py`
```

Comment:
tabs - but also for typescript that is wrong I think? The way we use typescript is with a front end. So maybe remove that. The person to ask is @maximpertsov

Reply:
Correct, typescript must be run in a web browser which requires some additional setup - we should definitely remove the `tsc ...` line and maybe have a link to the quick-start example instead. What do you think about that @npentrel ?

Reply:
sadly there is no one-liner for how to run the typescript SDK at the moment :(

Reply:
Thanks Maxim! In that case I do think we just need to remove that and link to quick start docs if available or tutorials

Reply:
Yeah, for now we can just point user to the vanilla example tutorial you wrote up, which is already on this page.

Reply:
@maximpertsov @npentrel thanks guys! will make these changes